tasks/sep/019-fp-holocene-upgrade/README.md

```diff
@@ -0,0 +1,37 @@
+# Holocene Hardfork Upgrade - OP Sepolia
+
+Status: READY TO SIGN
+
+## Objective
+
+Upgrades the Fault Proof contracts of OP Seplia for the Holocene hardfork.
+
+The proposal is soon posted to the governance forum. This is just the testnet upgrade.
+
+This upgrades the Fault Proof contracts in the
+[op-contracts/v1.8.0-rc.2](https://github.com/ethereum-optimism/optimism/tree/op-contracts/v1.8.0-rc.2) release.
```

Comment:
This should be rc.3, per https://github.com/ethereum-optimism/optimism/pull/13015#discussion_r1854157246

Reply:
[37066cc](https://github.com/ethereum-optimism/superchain-ops/pull/374/commits/37066cceddd0c0391b570ec077f8781629f8cb0c)